evaluator.go

```diff
@@ -1,7 +1,7 @@
 package goval
 
 import (
-	"github.com/maja42/goval/internal"
+	"./internal"
```

Comment:
@maja42 did you mean to do this?

Reply:
Yep, this breaks my code when I try to import it goval

Reply:
Oh, no that was an accident. I committed that on accident.

Reply:
np, just pushed a fix!